docs: review and edits of PR #2510 docs: manage workspaces with kubectl (#14)
dkwon17/che-docs: modules/end-user-guide/partials/proc_creating-workspaces.adoc

[id="creating-workspaces"]
= Creating workspaces

> Because workspaces on the {orch-name} cluster are represented by custom resources, and if your use case does not permit use of the {prod-short} dashboard, you can instead create workspaces with {orch-name} APIs by applying custom resources to the cluster.

Reviewer: ```suggestion
Because workspaces on the {orch-name} cluster are represented by custom resources, you can create workspaces with {orch-name} APIs by applying custom resources to the cluster instead of creating workspaces with the {prod-short} dashboard.
```

```suggestion
If your use case does not permit use of the {prod-short} dashboard, you can instead create workspaces with {orch-name} APIs by applying custom resources to the cluster. This is because {prod-short} workspaces are custom resources on the cluster level.
```

I find this sentence a bit awkward and thought of some suggestions, WDYT?

Author: Yes, you're right. Thank you for the suggestions.

Author: @dkwon17,  I've reworked your latter suggestion, WDYT?
```suggestion
If your use case does not permit use of the {prod-short} dashboard, you can create workspaces with {orch-name} APIs by applying custom resources to the cluster.
```

    preStart:
      - init-container-command
----
<1> Name of the `DevWorkspaceTemplate` custom resource. This name will be used to reference the `DevWorkspaceTemplate` custom resource within the `DevWorkspace` custom resource.
<2> User namespace, which is the target {orch-namespace} for the new workspace.
====

. To prepare the `DevWorkspace` custom resource, copy the contents of the target Git repository's devfile.
+
.Copied devfile contents with `schemaVersion: 2.1.0`
====
[source,yaml,subs="+quotes,+attributes"]
----
components:
  - name: tooling-container
    container:
      image: quay.io/devfile/universal-developer-image:ubi8-latest
----
====
+
TIP: For more details, see the link:https://devfile.io/docs/2.1.0/what-is-a-devfile[devfile v2 documentation].

> . Create a `DevWorkspace` custom resource definition that incorporates the copied devfile contents from the previous step.

Reviewer: ```suggestion
. Create a `DevWorkspace` custom resource definition that incorporates the copied devfile contents from the previous step. Devfile contents are set under the `spec.template` field of the `DevWorkspace` custom resource.
```

Author: @dkwon17, I reworked your suggestion above. WDYT? 
```suggestion
. Create a `DevWorkspace` custom resource definition, pasting the devfile contents from the previous step under the `spec.template` field.
```
